src/main/java/frc/robot/commands/shooter/ManualAdjust.java

```diff
@@ -0,0 +1,55 @@
+package frc.robot.commands.shooter;
+
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.Measure;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.RobotConfig.*;
+import frc.robot.subsystems.shooter.Shooter;
+
+public class ManualAdjust extends Command {
+  private final Shooter m_shooter;
+  private final AdjustType m_type;
+  private Measure<Angle> desiredAngle;
+  private Timer timer;
+
+  public ManualAdjust(Shooter shooter, AdjustType type) {
+    m_shooter = shooter;
+    m_type = type;
+    timer = new Timer();
+    addRequirements(m_shooter);
+  }
+
+  @Override
+  public void initialize() {
+    timer.start();
+    switch (m_type) {
+      case up:
+        desiredAngle = m_shooter.getCurrentAngle().plus(ShooterConfig.kAdjustAmountDegrees);
+        m_shooter.setAngle(desiredAngle);
+        break;
+      case down:
+        desiredAngle = m_shooter.getCurrentAngle().minus(ShooterConfig.kAdjustAmountDegrees);
+        m_shooter.setAngle(desiredAngle);
+        break;
+      default:
+        desiredAngle = m_shooter.getCurrentAngle();
+        m_shooter.setAngle(desiredAngle);
+        break;
+    }
+  }
+
+  @Override
+  public void execute() {
+    m_shooter.setFF(
+        Math.cos(Units.rotationsToRadians(m_shooter.getCurrentAngle().magnitude()))
```

Comment:
this is not using `Units` correctly. you need to do `m_shooter.getCurrentAngle().in(Units.Radians)`

Reply:
My vscode says Units.(any measure) doesn't exist, but I'll commit it. This was a workaround.